Makefile

```diff
@@ -17,10 +22,42 @@ COMMON_ARGS += --frontend-opt build-arg:GOLANG_VERSION=$(GOLANG_VERSION)
 COMMON_ARGS += --frontend-opt build-arg:SHA=$(SHA)
 COMMON_ARGS += --frontend-opt build-arg:TAG=$(TAG)
 
-all: kernel initramfs rootfs osctl-linux-amd64 osctl-darwin-amd64 test lint docs installer
+all: buildkitd builddeps kernel initramfs rootfs osctl-linux-amd64 osctl-darwin-amd64 test lint docs installer
+
+VPATH = $(PATH)
+
+.PHONY: builddeps
+builddeps: gitmeta buildctl
+
+gitmeta:
+	GO111MODULE=off go get github.com/autonomy/gitmeta
+
+buildctl:
+	go get github.com/moby/buildkit/cmd/buildctl
+
+.PHONY: buildkitd
+buildkitd:
+ifeq ($(CONTAINERSTOPPED),$(BUILDKIT_CONTAINER))
```

Comment:
Can we make this target optional? One of the benefits of buildkit is that it has excellent support for remote builds. Instead of running it on my laptop I spin it up on a home server and just point this build to it. It looks like this forces the run to be local only?

Reply:
Looks like I still need to account for this use case. I misread it at first and thought you were running locally as in making use of the socket.